common/djangoapps/student/forms.py

```diff
@@ -199,10 +199,12 @@ class AccountCreationForm(forms.Form):
         }
     )
     password = forms.CharField(
-        min_length=accounts_settings.PASSWORD_MIN_LENGTH,
+        min_length=password_min_length(),
+        max_length=password_max_length(),
         error_messages={
             "required": _PASSWORD_INVALID_MSG,
             "min_length": _PASSWORD_INVALID_MSG,
+            "max_length": _PASSWORD_INVALID_MSG,
```

Comment:
These guys are all the same? :(

Reply:
Yeah. "A valid password is required" -- this is the old forms, so I don't think this is shown on edx.org. But still. :-/